### PR TITLE
Set experimental.webpackBuildWorker=true to reduce build memory consumption ~10x

### PR DESCRIPTION
**Pull request by alebedev**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4721?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4721** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4721?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4716** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4716?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#4718](https://github.com/HedvigInsurance/racoon/pull/4718) <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4718?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/next.config.mjs [hunk 1]
```diff
@@ -62,11 +63,11 @@ const config = {
       },
     ]
   },
-  webpack(config) {
-    // Suppress known warnings from webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo complaining about PNP modules
-    config.infrastructureLogging = { level: 'error' }
-    return config
-  },
+  // webpack(config) {
```
Comment (Youakeem) [inline apps/store/next.config.mjs:66]: Should we keep this commented out?

**Review by Youakeem — APPROVED** [review]

File: apps/store/next.config.mjs (hunk 1 above)
Comment (alebedev) [reply apps/store/next.config.mjs:66]: That was a test change. I'll revert it for now, then will see if we still need it

**Review by guilhermespopolin — APPROVED** [review]

**Review by gustaveen — APPROVED** [review]

**Comment by alebedev** [comment]
### Merge activity

* **Sep 17, 7:49 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4721) rebased this pull request as part of a merge.

**Pull request merged**